src/systems.rs

```diff
@@ -266,7 +266,7 @@ pub fn process_input_system(
         }
     }
 
-    if !command || !*context_params.is_macos && ctrl && alt {
+    if !command && !win || !*context_params.is_macos && ctrl && alt {
```

Comment:
```suggestion
    if !command || !*context_params.is_macos && ctrl && alt {
```
I think adding `&& !win` to the condition makes it so that we no longer ignore character input on such hotkeys as `Ctrl + C` on Windows (i.e. on pressing `Ctrl + C` it'll both copy and paste the `c` character)

Reply:
The logic is a bit weird because it's negated, but it definitely works correctly as I tested it on Linux and it works the same on Windows.

It checks that both `Win` and `Ctrl` are not pressed before typing characters. I wouldn't want to type anything if pressing e.g `Win+Tab` or `Ctrl+C` .

Reply:
Yeah, my bad, I for some reason confused `win` (a modifier key) with the Windows platform. I surely haven't had enough sleep today 😄